### DISCUSSION TITLE
How implement Mask in field of form?

### DISCUSSION THREAD

**Discussion by Kakovisk** [Budibase/budibase#7488] [Help] [answered]
How implement Mask in field of form?

I woul like to implement mask for phone number.

**Comment by R2bEEaton** [comment, marked as answer]
Hi @Kakovisk,

I believe what you're looking for is a validation for the field to make sure that it's a phone number. You can use Regex for this. Set up a text field and set the "Validation" to the following...
![image](https://user-images.githubusercontent.com/34921506/187178724-4e4071d7-648c-4838-a713-aa6aab9e6a52.png)

The Regex I used was `^(\+\d{1,2}\s)?\(?\d{3}\)?[\s.-]?\d{3}[\s.-]?\d{4}$` which was taken from [this Stack Overflow answer](https://stackoverflow.com/a/16699507), but you can use whichever Regex you find or make to validate phone numbers in the way you want them. Make sure that if you have a save button for the form, that the first action is "Validate Form" so that it doesn't let users submit it without a valid phone number value.

Hope this helps! Reply with any questions or mark this as answered using the check mark above to help future readers.

**Reply by Kakovisk** [reply]
Thank You!!

**Reply by R2bEEaton** [reply]
No problem! Make sure to go ahead and mark the answer with the check mark above if it worked for you.
